.github/workflows/default.yaml

```
@@ -109,10 +109,35 @@ jobs:
           sudo apt-get --quiet=2 build-dep squid
           sudo apt-get --quiet=2 install linuxdoc-tools libtool-bin
 
+      - name: Install prerequisite MacOS packages
+        if: runner.os == 'macOS'
+        run: |
+          brew install \
+            automake coreutils cppunit gawk \
+            gnu-getopt gnu-sed grep libtool \
+            make openldap openssl@3 cyrus-sasl
+
       - name: Checkout sources
         uses: actions/checkout@v4
 
-      - run: ./test-builds.sh
+      - name: Run build on Linux
+        if: runner.os == 'Linux'
+        run: ./test-builds.sh
+
+      - name: Run build on MacOS
+        if: runner.os == 'macOS'
+        run: |
+          eval `brew shellenv`
+          export PKG_CONFIG_PATH="$HOMEBREW_PREFIX/lib/pkgconfig:$HOMEBREW_PREFIX/opt/openldap/lib/pkgconfig:$HOMEBREW_PREFIX/opt/cyrus-sasl/lib/pkgconfig"
```

I would split this into package-specific lines. The code becomes much more readable (and easier to maintain) that way IMO. For example:

```suggestion
          export PKG_CONFIG_PATH="$HOMEBREW_PREFIX/lib/pkgconfig"
          PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$HOMEBREW_PREFIX/opt/openldap/lib/pkgconfig"
          PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$HOMEBREW_PREFIX/opt/cyrus-sasl/lib/pkgconfig"
```

---

I don't think it helps much but it doesn't hurt either, so adopting a
even more readable version to advance the PR